lobby/src/main/java/org/triplea/lobby/server/controller/rest/ModeratorActionController.java

```diff
@@ -0,0 +1,84 @@
+package org.triplea.lobby.server.controller.rest;
+
+import static spark.Spark.before;
+import static spark.Spark.halt;
+import static spark.Spark.path;
+import static spark.Spark.post;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.triplea.lobby.server.db.Database;
+import org.triplea.lobby.server.db.HashedPassword;
+import org.triplea.lobby.server.db.UserController;
+
+import com.google.common.base.Splitter;
+
+import games.strategy.engine.lobby.server.userDB.DBUser;
+import spark.Request;
+import spark.Response;
+
+class ModeratorActionController {
```

Comment:
Can we apply the open-closed principle to this class?

Reply:
Sorry, bit too vague. I wonder if the granularity is fine enough to avoid everything moderator relator from landing in the same controller (Perhaps not a bad thing, but I wonder how large the controller class would wind up being)